tests/test.rs

```diff
@@ -538,3 +538,50 @@ fn test_integration_zip() {
         do_clean(path);
     }
 }
+
+#[test]
+fn test_integration_zip_dir() {
+    if cfg!(windows) {
+        println!("Integration tests still not supported under Windows.");
+        return;
+    }
+
+    let compilers = vec!["g++", "clang++"];
+
+    for compiler in compilers {
+        let is_llvm = compiler == "clang++";
+        let name = if is_llvm { "llvm" } else { "gcc" };
+        let path = &PathBuf::from("tests/basic_zip_dir");
```

Comment:
we should update https://github.com/mozilla/grcov/pull/150/files#diff-dcd61c41ae35cab6b9646368cca02e49R391 to ignore this too